### PR TITLE
Feature/issue 1012

### PR DESCRIPTION
**Pull request by BapBap7**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: __mocks__/antd/es/form/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  return [{
+    setFieldsValue: mockSetFieldsValue,
+    resetFields: mockResetFields,
+  }]
+};
+
+
+Form.useForm = useForm;
+Form.Item = Item;
+export default Form;
```
Comment (Rominos7) [inline __mocks__/antd/es/form/Form.tsx:35]: Insert empty line at the end of the file

File: __mocks__/antd/es/form/FormItem.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  required?: boolean;
+};
+
+type FormItemProps = {
+  name?: string;
+  label?: string;
+  className?: string;
+  rules?: Rule[];
+  valuePropName?: string;
+  // getValueFromEvent?: (...args) => any,
```
Comment (Rominos7) [inline __mocks__/antd/es/form/FormItem.tsx:13]: Do you need this comment?

File: __mocks__/antd/es/form/FormItem.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  valuePropName,
+  children,
+}: FormItemProps) => {
+  return (
+    <>
+      <div data-testid="form-item-label">{label}</div>
+      <div data-testid="form-item-name">{name}</div>
+      <div data-testid="form-item-class">{className}</div>
+      <div data-testid="form-item-value-prop-name">{valuePropName}</div>
+      {/* <div data-test-id="form-item-get-value-from-event">{getValueFromEvent}</div> */}
```
Comment (Rominos7) [inline __mocks__/antd/es/form/FormItem.tsx:31]: Do you need this comment?

File: __mocks__/fileMock.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+const mockedSvg = 'test-file-stub';
+export default mockedSvg;
```
Comment (Rominos7) [inline __mocks__/fileMock.ts:2]: Insert empty line at the end of the file

File: jest.config.ts [hunk 5]
```diff
@@ -1,52 +1,54 @@
...
+  moduleNameMapper: {
+    "\\.(css|less|sass|scss)$": "identity-obj-proxy",
+    "^@assets/(.*)$": "<rootDir>/src/assets/$1",
+    // PartnerModal specification for testing
+    "^antd/es/(.*)$": "<rootDir>/node_modules/antd/lib/$1",
+    // "^antd/(.*)$": "<rootDir>/node_modules/antd/$1",
+    // DEV_NOTE: Down below is the right solution for compiling files by path
+    // But we need to fix ALL typo errors before it could be compiled
+    // Temp solution - mocking modules.
+    // After fixing type error remove mocks or continue using them(as more simple solution)
```
Comment (Rominos7) [inline jest.config.ts:16]: Remove this comment if you dont need it anymore

**Review by Rominos7 — APPROVED** [review]

**Review by Adeodonne — APPROVED** [review]
Awesome work

**Review by Spoilsport0 — APPROVED** [review]

**Pull request merged**